README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
 
 ## Examples & Usages
 
-The following examples can give you an impression of what dataprep can do:
+The following examples can give you an impression of what Dataprep can do:
 
 * [Documentation: Connector](https://sfu-db.github.io/dataprep/user_guide/connector/connector.html)
 * [Documentation: EDA](https://sfu-db.github.io/dataprep/user_guide/eda/introduction.html)
 * [EDA Case Study: Titanic](https://sfu-db.github.io/dataprep/user_guide/eda/titanic.html)
 * [EDA Case Study: House Price](https://sfu-db.github.io/dataprep/user_guide/eda/house_price.html)
 
+### Connector
+
+Connector is an intuitive, open-source API wrapper that speeds up development by standardizing calls to multiple APIs as a simple workflow.
+
+
+Connector provides a simple programming interface to collect structured data from different Web APIs (e.g., Twitter API, Yelp Fusion API, Spotify API, DBLP API), making web data collection easy and efficient, without requiring advanced programming skills.
+
+
+Do you want to leverage the growing number of websites that are opening their data through public APIs?
+
+
+Connector is for you!  
+
+Let's check out the several benefits that Connector offers:
+
+* <ins>**A unified API:**</ins> You can fetch data using one or two lines of code to get data from many websites.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_main.gif"/></a>
+
+* <ins>**Auto Pagination:**</ins> Connector automatically does the pagination for you! Just specify the desired number of returned results (argument `_count`) without getting into unnecessary detail about a specific pagination scheme.
```

Comment:
Do you think it is worth defining pagination here @dovahcrow @peiwangdb ? Just to make sure our value is clearly understood by all audiences. Or even a hyperlink to a definition online.

Reply:
Done. The following sentence was included: "Do you want to invoke a Web API that could return a large result set and need to handle it through pagination? Connector automatically does the pagination for you!".